partcad/src/partcad/context.py

```
@@ -347,28 +347,28 @@ async def _import_all_recursive(self, project):
-                imports = list(filtered)
+                dependencies = list(filtered)
 
-            for prj_name in imports:
-                prj_conf = project.config_obj["import"][prj_name]
+            for prj_name in dependencies:
+                prj_conf = project.config_obj["dependencies"][prj_name]
 
                 if prj_conf.get("onlyInRoot", False):
                     next_project_path = "/" + prj_name
                 else:
                     next_project_path = get_child_project_path(project.name, prj_name)
```

Ruff (0.8.2)

`get_child_project_path` may be undefined, or defined from star imports

(F405)